iotdb-client/service-rpc/src/main/java/org/apache/iotdb/rpc/subscription/exception/SubscriptionIncompatibleHandlerException.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.rpc.subscription.exception;
+
+import java.util.Objects;
+
+public class SubscriptionIncompatibleHandlerException extends SubscriptionException {
+
+  public SubscriptionIncompatibleHandlerException(final String message) {
+    super(message);
+  }
+
+  public SubscriptionIncompatibleHandlerException(final String message, final Throwable cause) {
+    super(message, cause);
+  }
+
+  @Override
+  public boolean equals(final Object obj) {
+    return obj instanceof SubscriptionIncompatibleHandlerException
+        && Objects.equals(
+            getMessage(), ((SubscriptionIncompatibleHandlerException) obj).getMessage())
+        && Objects.equals(
+            getTimeStamp(), ((SubscriptionIncompatibleHandlerException) obj).getTimeStamp());
+  }
```

Comment:
hashCode

Reply:
Method 'hashCode()' is identical to its super method